deepchem/models/tensorgraph/models/graph_models.py

```diff
@@ -477,15 +486,17 @@ def default_generator(self,
 
 class GraphConvTensorGraph(TensorGraph):
 
-  def __init__(self, n_tasks, **kwargs):
+  def __init__(self, n_tasks, mode="classification", **kwargs):
```

Comment:
Tensorgraph already has a "mode" variable.  Should we remove it from the parent?